client/chat/channel.tsx

```diff
@@ -40,6 +40,9 @@ const UserListContainer = styled.div`
   width: 256px;
   flex-grow: 0;
   flex-shrink: 0;
+  overflow-y: auto;
+  overflow-x: hidden;
+  padding-bottom: 8px;
```

Comment:
I don't think this is the right place to add the padding. If you add it here, you end up with a 8px background-colored bar at the bottom that cuts off the content (the list of users). What we *actually* want is 8px of extra space at the end of the content (only visible when you scroll all the way to the bottom). To get that, you can probably add padding to the `VariableSizeList` (although I can't remember if that actually works with `react-window`, may need to be applied to some deeper element)